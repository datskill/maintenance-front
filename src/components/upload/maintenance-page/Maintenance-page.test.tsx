import * as React from "react";
import * as renderer from "react-test-renderer";
import MaintenancePage from "./Maintenance-page";

it("MaintenancePage renders correctly", () => {
  const tree = renderer.create(<MaintenancePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
