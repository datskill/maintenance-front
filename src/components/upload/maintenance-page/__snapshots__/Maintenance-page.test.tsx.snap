// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaintenancePage renders correctly 1`] = `
<div
  className="MuiGrid-root maintenance-site MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-between"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12"
  >
    <p
      className="style-text"
    >
      Le site est actuellement en maintenance
    </p>
  </div>
</div>
`;
